### PR TITLE
445 - Change tab style

### LINKED ISSUE #445: Change tab style
### Context

Change that came up during the design file refactor

### Problem or idea
Change the existing tab style for the style below: 
Screenshot 2023-09-19 at 14 47 23

The component is in this frame:
https://www.figma.com/file/FqWPogwIivEoiyuF5C6eRs/scpca-styles-components?type=design&node-id=35%3A502&mode=design&t=VsXRwzkckYSiOV8g-1

### Solution or next step
Tagging @davidsmejia for next steps

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number

Closes #445

## Purpose/Implementation Notes

Based on the design update, changed the tab style on the project page.

@dvenprasad please review the latest UI [here](https://scpca-portal-e0tsmzxvv-ccdl.vercel.app).

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

### CODE REVIEW

File: client/src/theme/tabs.js [hunk 1]
```diff
@@ -23,11 +30,11 @@ export default {
       }
 
       > button[aria-expanded="true"] {
-        font-weight: 700;
-        border-left: 1px solid ${theme.global.colors.border.light};
-        border-top: 1px solid ${theme.global.colors.border.light};
-        border-right: 1px solid ${theme.global.colors.border.light};
-        border-bottom: 1px solid ${theme.global.colors.background.light};
+        border-bottom: 2px solid ${theme.global.colors.brand.light};
```
Comment (davidsmejia) [inline client/src/theme/tabs.js:33]: Another way to pull colors from the theme without defining them as hex values would be to use the `normalizeColor` util. You can see it used here: https://github.com/AlexsLemonade/scpca-portal/blob/c285a5b6445be16b3d47a0b19317c98614771b46/client/src/theme/select.js#L28

I know that we use this style of grabbing colors elsewhere but ideally we shouldn't be converting colors to hex in order to use them this way.

```suggestion
        border-bottom: 2px solid ${normalizeColor('brand', props.theme)};
```

**Review by davidsmejia — APPROVED** [review]
LGTM left one comment for consideration

**Review by dvenprasad — APPROVED** [review]
🐈 🐈 🐈  Looking good!!

File: client/src/theme/tabs.js (hunk 1 above)
Comment (nozomione) [reply client/src/theme/tabs.js:33]: Let's reorganize the theme files at some point (e.g., create a separate theme file for `colors`) and specify our preferred approach to pulling colors (using `normalizeColor` or dot notation), appreciate your tip, David.

**Pull request merged**